weave/Scripts/Utils/ActionConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace weave.Utils;
+
+public static class ActionConstants
+{
+    public const string GamepadJoinAction = "gamepad_join";
```

Comment:
Is this and gamepad_leave buttons that are mapped by the gamepad? Or where is the corresponding button on the gamepad defined?

Reply:
These are my actions : -) (in the godot project)

Binded to A and B or something like that right now